modules/security/pages/password-policy.adoc

```diff
@@ -1,32 +1,121 @@
 
 == Enable password policy
-If the password policy is enabled, you cannot use passwords that do not satisfy the policy when creating or altering a new user password or ACL password.
+If the password policy is enabled, you cannot use passwords that do not satisfy the policy when creating or altering a new user password.
 Enabling the password policy does not prevent existing users with passwords that do not meet the requirements from logging in.
 
 To enable the password policy, run the following commands:
 
 [.wrap,console]
 ----
 $ gadmin config set Security.UserPasswordPolicy.Enable true <1>
-$ gadmin config set Security.ACLPasswordPolicy.Enable true  <2>
 $ gadmin config apply
 $ gadmin restart gsql
 ----
 <1> Setting `Security.UserPasswordPolicy.Enable` to true applies the password policy to passwords for TigerGraph database users.
-<2> Setting `Security.ACLPasswordPolicy.Enable` to true applies the password policy to ACL passwords.
\ No newline at end of file
+
+== How to config the password policy
+The configuration about the complexity rules as followings, we use “gadmin config” to set the value. 
+
+|===
+| Policy rule | Description | Default | Value Range
+
+| `Security.UserPasswordPolicy.Enable`
+| Enable/disable the password policy
+| false
+| true/false
+
+| `Security.UserPasswordPolicy.MinLength`
+| Min length
+| 8
+| [1,128]
+
+| `Security.UserPasswordPolicy.UppercaseLetterRequired`
+| Require at Least One Uppercase Character
+| true
+| true/false
+
+| `Security.UserPasswordPolicy.LowercaseLetterRequired`
+| Require at Least One Lowercase Character
+| true
+| true/false
+
+| `Security.UserPasswordPolicy.DigitRequired`
+| Require at Least One Digit
+| true
+| true/false
+
+| `Security.UserPasswordPolicy.SpecialCharacterRequired`
+| Require at Least One special character .
+| true
+| true/false
+
+| `Security.UserPasswordPolicy.ExpirationDay`
+| How long the password remains valid after it has been updated
+| 90
+| [1, 2147483647]
+
+| `Security.UserPasswordPolicy.PasswordReuseThreshold`
+| What is the minimum number of unique passwords required before a previously used password can be reused
+| 5
+| [1, 20]
+|===
+
+
+Follows are the examples of passworld policy configuration:
+
+[.wrap,console]
+----
+$ gadmin config set Security.UserPasswordPolicy.PasswordReuseThreshold 2
+$ gadmin config set Security.UserPasswordPolicy.MinLength 8
+$ gadmin config set Security.UserPasswordPolicy.UppercaseLetterRequired true
+$ gadmin config set Security.UserPasswordPolicy.LowercaseLetterRequired true
+$ gadmin config set Security.UserPasswordPolicy.DigitRequired true
+$ gadmin config set Security.UserPasswordPolicy.SpecialCharacterRequired true
+$ gadmin config set Security.UserPasswordPolicy.ExpirationDay 2
+$ gadmin config set Security.UserPasswordPolicy.PasswordReuseThreshold 2
+$ gadmin config apply
+$ gadmin restart gsql
+----
+
+If we configure the values as above, here are some usage examples:
```

Comment:
Same as above. Let's avoid the usage of we and use direct phrasing.
Change to : Based on the above configuration, here are some usage examples: